Makefile

```diff
@@ -14,3 +14,8 @@ clean:
 install:
 	@echo "Installing dependencies..."
 	@pip install antlr4-tools
+
+test_cases:
+	@echo "Generating test cases..."
+	@grammarinator-process gqlex.g4 -o tests/fuzzer/ --no-actions
```

Comment:
Based on the error message, I think you want `-r document` here, too.

Reply:
What error message did you get?
I added venv with its deps. so now `make` should run the same for any env

Reply:
The farthest rule from 'NEGATIVE_SIGN' is 'NEGATIVE_SIGN' (0 steps).

42 rule(s) unreachable from 'NEGATIVE_SIGN': 'EOF', 'NONZERO_DIGIT', 'DIGIT', 'LETTER', 'NAME_START', 'NAME_CONTINUE', 'SLASH', 'DOUBLE_SLASH', 'DOTS', 'RANGE_OPEN', 'RANGE_CLOSE', 'COLON', 'EQUALS', 'LBRACK', 'RBRACK', 'LPAREN', 'RPAREN', 'COND_EQ', 'COND_NE', 'COND_GT', 'COND_GE', 'COND_LT', 'COND_LE', 'COND_AND', 'COND_OR', 'WS', 'POSITIVE_INT', 'NEGATIVE_INT', 'NAME', 'document', 'pathExpression', 'rootPathExpression', 'rangePathExpression', 'elementSelectionExpression', 'pathElement', 'conditionOperator', 'conditionLogicalOperator', 'conditionExpression', 'nameValue', 'intValue', 'rangeEndIntValue', 'rangeStartIntValue'

Reply:
You have `-r document` for the second commend but not for the first one. I think you should have it in both.

Reply:
done